### ISSUE TITLE
ebiten: Freeze (or dead lock) at networking on browsers

### ISSUE THREAD

**Issue by hajimehoshi** [hajimehoshi/ebiten#1161]
https://github.com/odencat/splash-ebiten/tree/e256447cee5c2fe0aaecdfa1edb0ea508aa005fb

```
GOOS=js GOARCH=wasm go build -o main.wasm && go run server/server.go 
```

With Go 1.14.3, this application freezes on browsers.

With Go 1.13.9, I found this error:

```
fatal error: all goroutines are asleep - deadlock!
wasm_exec.js:41 
wasm_exec.js:41 goroutine 1 [chan receive]:
wasm_exec.js:41 github.com/hajimehoshi/ebiten/internal/uidriver/js.(*UserInterface).Run(0x464620, 0x11d320, 0x46ff00, 0x0, 0x717c0)
wasm_exec.js:41 	/Users/hajimehoshi/go/pkg/mod/github.com/hajimehoshi/ebiten@v1.11.1/internal/uidriver/js/ui.go:412 +0x6
wasm_exec.js:41 github.com/hajimehoshi/ebiten.runGame(0x11afe0, 0x828f00, 0x0, 0xd249e8, 0x80a2c0)
wasm_exec.js:41 	/Users/hajimehoshi/go/pkg/mod/github.com/hajimehoshi/ebiten@v1.11.1/run.go:292 +0x6
wasm_exec.js:41 github.com/hajimehoshi/ebiten.RunGame(0x11b060, 0x80a2c0, 0x0, 0x0)
wasm_exec.js:41 	/Users/hajimehoshi/go/pkg/mod/github.com/hajimehoshi/ebiten@v1.11.1/run.go:283 +0xb
wasm_exec.js:41 main.main()
wasm_exec.js:41 	/Users/hajimehoshi/ebiten-games/splash-ebiten/example/ebiten/main.go:318 +0x4
wasm_exec.js:41 
wasm_exec.js:41 goroutine 8 [select]:
wasm_exec.js:41 github.com/hajimehoshi/ebiten/internal/uidriver/js.(*UserInterface).loop.func3(0x830180, 0x464620, 0x830120)
wasm_exec.js:41 	/Users/hajimehoshi/go/pkg/mod/github.com/hajimehoshi/ebiten@v1.11.1/internal/uidriver/js/ui.go:235 +0x9
wasm_exec.js:41 created by github.com/hajimehoshi/ebiten/internal/uidriver/js.(*UserInterface).loop
wasm_exec.js:41 	/Users/hajimehoshi/go/pkg/mod/github.com/hajimehoshi/ebiten@v1.11.1/internal/uidriver/js/ui.go:217 +0xf
```

**Comment by hajimehoshi** [comment]
CC @wasedaigo

**Comment by hajimehoshi** [comment]
https://github.com/golang/go/issues/25902#issuecomment-397628566

Now `requestAnimationFrame` in Ebiten doesn't create a new goroutine, and that's why.

**Comment by hajimehoshi** [comment]
This was fixed on Go 1.13, but not Go 1.14 yet...
